src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -509,6 +509,11 @@ export default {
     isFollowed() {
       return this.walletFollowees?.includes(this.iscnOwner) || false;
     },
+    controlBarPriceLabel() {
+      return this.nftIsNFTBook
+        ? this.nftBookFeaturedPrice
+        : formatNumberWithLIKE(this.NFTPrice);
```

Comment:
I think a better way is to check if book store price exists, if not show LIKE price if exists.